pkgs/top-level/all-packages.nix

```
@@ -11463,7 +11463,14 @@ in
 
   getdns = callPackage ../development/libraries/getdns { };
 
-  gettext = callPackage ../development/libraries/gettext { };
+  gettext-tools = callPackages ../development/tools/gettext-tools { };
+
+  gettext-runtime = callPackages ../development/libraries/gettext-runtime { };
+
+  libtextstyle = callPackage ../development/libraries/libtextstyle { };
+
+  # Ease transition between one-package gettext to split packages (tools - runtime)
+  gettext = gettext-runtime;
```

This is the most common use case.

```suggestion
  gettext = gettext-tools;
```